src/main/java/io/vertx/ext/hawkular/VertxHawkularOptions.java

```diff
@@ -241,4 +241,15 @@ public VertxHawkularOptions setBatchDelay(int batchDelay) {
     this.batchDelay = batchDelay;
     return this;
   }
+
+  @Override
+  public boolean isEnabled() {
+    return super.isEnabled();
+  }
+
+  @Override
+  public MetricsOptions setEnabled(boolean enable) {
```

Comment:
Maybe this should be `public VertxHawkularOptions setEnabled(boolean enable) {` sot that users can write `VertxHawkularOptions options = new VertxHawkularOptions().setEnabled(true).setHost(xxx);` ?